test/unit/query/builder.js

```diff
@@ -2238,6 +2238,26 @@ describe("QueryBuilder", function() {
     });
   });
 
+  it("update method with returning on oracle", function() {
+    testsql(qb().from('users').where('id', '=', 1).update({email: 'foo', name: 'bar'}, '*'), {
+      oracle: {
+        sql: 'update "users" set "email" = ?, "name" = ? where "id" = ? returning ROWID into ?',
+        // bindings: ['foo', 'bar', 1]
+        bindings: function(bindings) {
+          expect(bindings.length).to.equal(4);
+          expect(bindings[0]).to.equal('foo');
+          expect(bindings[1]).to.equal('bar');
+          expect(bindings[2]).to.equal(1);
+          expect(bindings[3].toString()).to.equal('[object ReturningHelper:*]');
```

Comment:
Oh, I should get rid of that comment on 2245. The bindings for `returning ROWID into ?` should link to the OutParam obj used returningSql helper (following the pattern for the insert method returning tests https://github.com/tgriesser/knex/blob/master/test/unit/query/builder.js#L2329)